### PR TITLE
Breakdown section added

### PR DESCRIPTION
**Pull request by kiraigg**

### CODE REVIEW

File: client/src/components/NewsBlock/index.jsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import hover_arrow from '../../assets/img/news-hover-arrow.svg'
+import { Link } from 'react-router-dom'
+
+export default function NewsBlock({ newsBlock }) {
```
Comment (InnaKutsik) [inline client/src/components/NewsBlock/index.jsx]: Why not const and arrow function without return?

File: client/src/components/Home/index.jsx [hunk 2]
```diff
@@ -1,18 +1,27 @@
...
 import { MostPopularComments } from '../MostPopularComments'
+import { Breakdown } from '../Breakdown'
 import MostPopular from '../../assets/data/MostPopularNewsList'
 import MostComments from '../../assets/data/MostCommentsNewsList'
+import BreakDown from '../../assets/data/BreakdownNewsList'
+import SingleNews from '../../assets/data/SingleNews'
 
 export const Home = () => (
   <div className='home-page-wrapper'>
+    <div className='breakdown-container'>
```
Comment (InnaKutsik) [inline client/src/components/Home/index.jsx]: Why do we need this wrapper?

File: client/src/components/NewsOfTheDays/index.jsx [hunk 3]
```diff
@@ -1,10 +1,11 @@
 import React from 'react'
-import { Link } from 'react-router-dom'
+import {Link} from 'react-router-dom'
 
 export const NewsOfTheDays = (props) => {
-  const { image, title, text, photo_courtesy, full_news } = props
+  const {image, title, text, photo_courtesy, full_news} = props
+
   function showSeeMoreButton(text, full_news) {
```
Comment (InnaKutsik) [inline client/src/components/NewsOfTheDays/index.jsx]: Why not arrow function without return?

File: client/src/components/SingleNews/index.jsx [hunk 4]
```diff
@@ -2,14 +2,16 @@ import React from 'react'
 import hover_arrow from '../../assets/img/news-hover-arrow.svg'
 import { Link } from 'react-router-dom'
 
-export default function NewsBlock({ newsBlock }) {
+export default function SingleNews({ singleNews }) {
```
Comment (InnaKutsik) [inline client/src/components/SingleNews/index.jsx]: Why not arrow function without return?

Comment (kiraigg) [reply client/src/components/SingleNews/index.jsx]: reworked

File: client/src/components/NewsOfTheDays/index.jsx (hunk 3 above)
Comment (kiraigg) [reply client/src/components/NewsOfTheDays/index.jsx]: reworked

File: client/src/components/Home/index.jsx (hunk 2 above)
Comment (kiraigg) [reply client/src/components/Home/index.jsx]: for interrupting the weights of the selectors

File: client/src/components/NewsBlock/index.jsx (hunk 1 above)
Comment (kiraigg) [reply client/src/components/NewsBlock/index.jsx]: reworked

**Pull request merged**